helpers/utils.py

```diff
@@ -9,10 +9,18 @@
 import logging
 import io
 from tempfile import NamedTemporaryFile
+import yaml
 
 USER_HOME_DIR_ENT_IDX = 5
 EXIT_CODE_BAD_KUBECONFIG = -100
 EXIT_CODE_BAD_CONF_ARG = -200
+EXIT_CODE_GET_SECRET_FAILED = -300
+EXIT_CODE_PUT_SECRET_ENV_FILE_FAILED = -400
+EXIT_CODE_PUT_SECRET_LITERAL_FAILED = -500
+EXIT_CODE_DEL_SECRET_FAILED = -600
+EXIT_CODE_GET_K8S_PROPS_FAILED = -700
+EXIT_CODE_GET_PRIMARY_RESOURCES_FAILED = -800
+EXIT_CODE_SET_PRIMARY_RESOURCES_FAILED = -900
```

Comment:
We should probably build an enum object with all of these, for better readability and grouping. But now that I'm thinking, we should also probably tackle these with exceptions and exception handlers. 

Anyhow, it is probably not worth addressing this in this PR (and it goes beyond this scope), but we need to do some thinking and design on how we want to handle exceptions at scale. Just mentioning here, so that we start to think about this improvement